src/Faker/Provider/Base.php

```diff
@@ -263,7 +263,7 @@ public static function shuffle($arg = '')
      * @param array $array The set to shuffle
      * @return array The shuffled set
      */
-    public static function shuffleArray($array = array())
+    public static function shuffleArray(array $array = array())
```

Comment:
Changing the signature of a public or protected function on a non-final class is breaking for anyone extending the class.